ra/ra_test.go

```diff
@@ -1186,23 +1191,30 @@ func TestNewCertificate(t *testing.T) {
 
 	// Check that we fail if the CSR signature is invalid
 	ExampleCSR.Signature[0]++
```

Comment:
Just modify the Raw bytes here, instead of modifying the signature. Modifying and single raw byte should cause signature verification to fail (although modifying the wrong raw byte may cause deserialization to fail first).